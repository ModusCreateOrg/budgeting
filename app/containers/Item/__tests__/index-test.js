import React from 'react';
import renderer from 'react-test-renderer';
import ItemHeader from '../itemHeader';

it('renders correctly', () => {
  const mockData = {
    description: "Trader Joe's food",
    inflowPercentage: 5.5,
    outflowPercentage: 2.5,
  };
  const tree = renderer
    .create(
      <ItemHeader name={mockData.description} inflow={mockData.inflowPercentage} outflow={mockData.outflowPercentage} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
