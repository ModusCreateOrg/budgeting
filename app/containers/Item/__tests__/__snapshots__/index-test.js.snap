// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="itemHeader"
>
  <h1>
    Trader Joe's food
  </h1>
  <div>
    <div
      className="metaText"
    >
      <span
        className="green"
      >
        + 
        5.50
      </span>
    </div>
    <div
      className="metaText"
    >
      <span
        className="red"
      >
        - 
        2.50
      </span>
    </div>
  </div>
</div>
`;
